util-app/src/main/scala/com/twitter/app/GlobalFlag.scala

```diff
@@ -117,9 +117,18 @@ abstract class GlobalFlag[T] private[app] (
 object GlobalFlag {
 
   private[app] def get(flagName: String): Option[Flag[_]] = {
-    def tryMethod(className: String, methodName: String): Option[Flag[_]] =
+    val className = if (!flagName.endsWith("$")) flagName + "$" else flagName
+    try {
+      get(Class.forName(className))
+    } catch {
+        case _: ClassNotFoundException | _: NoSuchFieldException | _: IllegalArgumentException =>
```

Comment:
Do we need the NoSuchFieldException & IllegalArgumentException here?

Reply:
@yufangong Thanks, fixed it